server/router.go

```diff
@@ -32,6 +32,7 @@ func Init(e *gin.Engine) {
 	})
 	g.GET("/favicon.ico", handles.Favicon)
 	g.GET("/robots.txt", handles.Robots)
+	g.GET("/static/manifest.json", handles.ManifestJSON)
```

Comment:
```suggestion
	g.GET("/manifest.json", handles.ManifestJSON)
```

改到根目录，然后前端也跟着改一下，应该就不会路由冲突了

Reply:
> 前端那边不用动，后端处理好就行

也可以

前端那边写的 

```html
  <link href="/static/manifest.json" rel="manifest" />
```

是固定的路径，我觉得可以给它替换掉

Reply:
我觉得还是像之前那样加个路由，然后替换 `/static/manifest.json` 字符串比较好

Reply:
前端改的话不好搞，要根据用户请求动态更新，而且要等DOM加载完，可能会有时序问题

Reply:
我现在在后端写了动态修改了index.html替换前端/static/manifest.json路径，前端不动就好，如果前端改了一样会撞车，而且DOM加载完才能动态变更，时序问题修起来更麻烦

Reply:
还是得改掉前端的路径，这个link被 vite-plugin-dynamic-base 给处理了

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/e138db61-2ce3-49e5-a348-5427521b550c" />